### PR TITLE
set DataFolder to match 3.6 ProbeConfig (SOFTWARE-4596)

### PR DESCRIPTION
**Pull request by edquist**

### CODE REVIEW

File: base/usr/local/bin/configure-nonroot-gratia.py [hunk 1]
```diff
@@ -30,6 +30,7 @@ def main():
     probe_config.attrib['Lockfile'] = '/var/lock/condor-ce/gratia.lock'
     probe_config.attrib['WorkingFolder'] = '/var/lib/condor-ce/'
     probe_config.attrib['DataFolder'] = '/var/lib/condor-ce/gratia/data/'
+    probe_config.attrib['CondorCEHistoryFolder'] = '/var/lib/condor-ce/gratia/data/'
```
Comment (brianhlin) [inline base/usr/local/bin/configure-nonroot-gratia.py]: Is setting these two to the same dir safe?

Comment (edquist) [reply base/usr/local/bin/configure-nonroot-gratia.py]: probably safe but also not really correct.  i've made some updates per our discussion

**Review by brianhlin — APPROVED** [review]
LGTM. We had a discussion on Zoom about dumping the logs into `/var/log/condor-ce/gratia/`, which would also need to be created with the right ownership

**Comment by edquist** [comment]
> LGTM. We had a discussion on Zoom about dumping the logs into `/var/log/condor-ce/gratia/`, which would also need to be created with the right ownership

Added.

**Pull request merged**